aw_server/server.py

```diff
@@ -90,14 +90,18 @@ def _config_cors(cors_origins: List[str], testing: bool):
 
 # Only to be called from aw_server.main function!
 def _start(
-    storage_method,
-    host: str,
-    port: int,
-    testing: bool = False,
-    cors_origins: List[str] = [],
+        storage_method,
+        host: str,
+        port: int,
+        testing: bool = False,
+        cors_origins: List[str] = [],
+        custom_static: Dict[str, str] = dict()
```

Comment:
Not sure why indentation changed?

```suggestion
    storage_method,
    host: str,
    port: int,
    testing: bool = False,
    cors_origins: List[str] = [],
    custom_static: Dict[str, str] = dict()
```